crates/containerd-shim-wasm/src/sys/mod.rs

```diff
@@ -0,0 +1,18 @@
+#[cfg(unix)]
+mod unix;
+#[cfg(windows)]
+mod windows;
+
+pub mod metrics {
```

Comment:
I like the naming of these modules!

Reply:
could you take a look again? I change the module structure in https://github.com/containerd/runwasi/pull/238#discussion_r1300764238.  Do you feel we need those for organization or does the new structure work

Reply:
The new structure works

Reply:
You could still keep the module structure.
You coud create a folder for each system, and place files for each module.